play.pokemonshowdown.com/src/panel-battle.tsx

```diff
@@ -800,6 +849,20 @@ class BattlePanel extends PSRoomPanel<BattleRoom> {
 				</ChatLog>
 				<ChatTextEntry room={room} onMessage={this.send} onKey={this.onKey} left={0} />
 				<ChatUserList room={room} top={this.battleHeight} minimized />
+				<button
+					class="icon button"
+					name="openBattleOptions"
+					title="Options"
```

Comment:
The button itself already says "Battle Options". The title "Options" feels redundant with that.